api/src/main/resources/db/migration/V7__CreateImagesTable.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS Images (
+    id SERIAL NOT NULL PRIMARY KEY,
+    file_name VARCHAR(200),
+    file_location VARCHAR(200)
+);
```

Comment:
I think both the images and tags table will need foreign keys linking them to the memes table, but I didn't include them right now since we aren't saving a meme yet